### ISSUE TITLE
RTIMULib copy should be optional

### ISSUE THREAD

**Issue by teuteuguy** [RPi-Distro/python-sense-hat#88]
Copying the RTIMULib to the home folder requires access to the home folder, which in some cases, ie. running as root can become complex.

This should be a setting on load of the lib, to choose whether or not to copy the RTIMULib file or not.

Event, be able to provide path to file directly.

**Comment by XECDesign** [comment]
In general, you should be guaranteed access to your home directory, if nowhere else. But yes, flexibility wouldn't hurt.

**Comment by J-Hurtta** [comment]
Indeed. I'm trying to use python-sense-hat on on a limited access case (Home Assistant on Hass.io).

As the RTIMULib is not listed as a dependency, I can't install it on my device and thus cannot use my sense hat.

**Comment by rzr** [comment]
or at least  requierement should be mentionned somewhere
